src/reporter/plugin-host.ts

```diff
@@ -0,0 +1,210 @@
+import chalk, { Chalk } from 'chalk';
+import indentString from 'indent-string';
+import {
+    identity,
+    escape as escapeHtml,
+    assignIn
+} from 'lodash';
+
+import moment from '../utils/moment-loader';
+import OS from 'os-family';
+import { wordWrap, removeTTYColors } from '../utils/string';
+import getViewportWidth from '../utils/get-viewport-width';
+import { DIFF_COLORS } from '../utils/diff/colors';
+import { Moment } from 'moment';
+import ReporterStreamController from '../runner/reporter-stream-controller';
+import { Stream } from 'stream';
+import TestRunErrorFormattableAdapter from '../errors/test-run/formattable-adapter';
+
+// NOTE: we should not expose internal state to
+// the plugin, to avoid accidental rewrites.
+// Therefore we use symbols to store them.
+
+/*global Symbol*/
```

Comment:
Do we really need this `/*global Symbol*/` comment? I guess it was left since times when Node didn't support ES6 natively.